api/src/main/java/org/openmrs/module/ugandaemr/api/impl/UgandaEMRServiceImpl.java

```diff
@@ -2045,9 +2045,6 @@ public void installCommonMetadata() {
         MetadataDeployService deployService = Context.getService(MetadataDeployService.class);
         try {
             log.info("Installing metadata");
-            log.info("Installing commonly used metadata");
-            deployService.installBundle(Context.getRegisteredComponents(CommonMetadataBundle.class).get(0));
```

Comment:
@slubwama why disabling installation of METADATA are you looking at doing this in the content package??